scripts/o3de/o3de/enable_gem.py

```diff
@@ -202,6 +228,11 @@ def add_parser_args(parser):
     parser.add_argument('-egf', '--enabled-gem-file', type=pathlib.Path, required=False,
                         help='The cmake enabled_gem file in which the gem names are specified.'
                         'If not specified it will assume enabled_gems.cmake')
+    group = parser.add_mutually_exclusive_group(required=False)
+    group.add_argument('-f', '--force', required=False, action='store_true', default=False,
+                       help='Bypass version compatibility checks')
+    group.add_argument('-c', '--check', required=False, action='store_true', default=False,
+                       help='Checks if this gem can be enabled for the specified project, but does not actually enable.')
```

Comment:
The check option is similar to the same vein of a [dry-run option](https://dpb.bitbucket.io/always-use-dry-run-options-if-possible.html)

Reply:
I'd be fine switching the arg name to `--dry-run` if that's more standard - thoughts?

Reply:
Yeah --dry-run is normally the more standard option.
`git` commands have a lot of dry run options, [rsync](https://linux.die.net/man/1/rsync) has one

Reply:
I went ahead and switched the argument name to `dry-run`